Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/CosmosServerlessSparkBatchJob.java

```diff
@@ -276,23 +284,20 @@ public String getJobUuid() {
                 .filter(sparkBatchJob -> sparkBatchJob.properties() != null
                         && StringUtils.isNotEmpty(sparkBatchJob.properties().livyServerAPI()))
                 .map(sparkBatchJob -> sparkBatchJob.properties().livyServerAPI())
-                .map(url -> url + "?adlaAccountName=" + getAccount().getName())
```

Comment:
`adlaAccountName` is not required?

Reply:
Add `TODO: need to test to see if adlaAccountName is necessary`